Ginger/GingerCoreNET/DataSource/LiteDB.cs

```diff
@@ -719,9 +725,12 @@ public void RunQuery(string query, int LocateRowValue, string DSTableName, bool
             //Nextavailable
             else if (NextAvai)
             {
-                DataTable datatble = GetQueryOutput("db." + DSTableName + ".find GINGER_USED = \"False\" limit 1");
-                row = datatble.Rows[LocateRowValue];
-                string rowID = Convert.ToString(row["GINGER_ID"]);
+                DataTable datatble = GetQueryOutput("db." + DSTableName + ".select $.GINGER_ID where GINGER_USED = \"False\" limit 1");
+                row = datatble.Rows[0];
+                string rowID = row[0].ToString();
+                //DataTable datatble = GetQueryOutput("db." + DSTableName + ".find GINGER_USED = \"False\" limit 1");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4997035)